tests/test_target_suitecrm.py

```diff
@@ -264,11 +276,55 @@ def test_users_create_admin_group(basic_config, suitecrm_server):
     )
     diff = target.calculate_difference()
     target.users_create(diff)
-    users = server.search_by_type("User")
+    return server
```

Comment:
I think what this code is doing is creating a server, using Lifecycle's TargetSuiteCRM to populate it, then returning it for later user. I don't think fixtures should be doing this, as they're meant to be setting up the things we test, rather than using the things we test.

It would be more appropriate to use the methods provided by MockSuiteCRMServer (the class that `server` is an object of) to populate it. 

e.g. use `MockSuiteCRMServer.create_user()` to create foobar and bazquux, then `MockSuiteCRMServer.create_record()` to create the admin group, then `MockSuiteCRMServer.create_relationship()` to add the group to the users.

This doesn't let us deduplicate any work, since the purpose of test_users_add_admin is to test that TargetSuiteCRM can create users in the admin group.